libs/mstable/flatcoin/src/state.ts

```diff
@@ -174,9 +174,17 @@ export const {
               : {
                   orderData: data[3]?.['orderData'] as string,
                   type: data[3]?.['orderType'] as number,
-                  keeperFee: (
-                    data[3]?.['keeperFee'] as BigNumberish
-                  ).toString(),
+                  keeperFee: new BigNumber(
```

Comment:
What do you think of storing such values not as `string` but as `BigDecimal`? I tried this approach yesterday and it is pretty convenient, I can refactor most places if you like the idea